src/fastfilter.h

```diff
@@ -23,17 +23,25 @@ class uint256;
  *
  * This class can be used anywhere a Bloom filter is used so long as the input data is random.
  *
- * This class is thread-safe in the sense that simultaneous calls to insert and contains will not crash,
+ * If NUM_HASH_FNS is 16 and FILTER_SIZE is >= 64k all bits in the uint256 input data will be used to set bits in
+ * the filter.  If these fields are set to lower numbers, fewer bits may be used (although in the NUM_HASH_FNS case
+ * execution will be faster).  Therefore, if this structure is used in an application that accepts externally created
```

Comment:
I think this should be further clarified a bit. It is not only that 'all bits are used' it is that they are **re**used in a straight-forward redundant way without further mixing when the filter size is >64k, which will very likely drive up false positive rates above what would be expected by a proper bloom approach. The user should know this.

Reply:
done